cryptol-remote-api/src/CryptolServer/Sat.hs

```diff
@@ -124,25 +215,25 @@ instance ToJSON ProveSatResult where
                   | res <- xs
                   ]
                 ]
+  toJSON (OfflineSMTQuery smtlib) =
+    JSON.object [ "result" .= ("offline" :: Text)
```

Comment:
Don't forget to update the `resultFieldDescription` with this new information.

Reply:
Thanks!